app/templates/components/job-detail-body.hbs

```diff
@@ -0,0 +1,26 @@
+{{#if job.isNew}}
+  <p>To start running the job, you must provide an authorization code below.</p><p>To request a code, please contact the {{share-group-contact job.shareGroup}} group.</p>
+{{else if job.isAuthorized}}
+  <p>The job will run on a virtual machine in the Bespin cloud infrastructure. The virtual machine will stage your data in, run the workflow, and upload the results to Duke Data Service.</p>
+  <p>You can start the job by clicking <strong>Start</strong>.</p>
+  {{job-authorize job}}
```

Comment:
I think this needs to be moved to job.isNew

Reply:
Yep it needs to be there. Actually I meant to have it in both. When a job already has authorization, the `{{job-authorize}}` component renders an un-editable input field showing the token